piton/verif/diag/assembly/include/riscv/ariane/crt.S

```diff
@@ -162,6 +162,20 @@ _start:
   sll sp, sp, STKSHIFT
   add sp, sp, tp
 
+//#ifdef PITON_NOC_IPI
```

Comment:
Could we add back this ifdef?

Reply:
IIRC there was an issue with the wakeup interrupt without it, but maybe there was a fix in the meantime?

Reply:
Reverted